### ISSUE TITLE
The lighten tool's cursor icon is misleading

### ISSUE THREAD

**Issue by captainbrosset** [juliandescottes/piskel#351]
I had selected the lighten tool from the toolbar like 10 minutes ago and had been doing something else in the meantime. So I forgot about it. When I came back, I was sure I had the pen selected, but trying to draw wouldn't work correctly, it wouldn't use my current color. That's because I really didn't have the pen selected, but the lighten tool instead.

Maybe it's just me, but I think that the confusion came from the cursor icon. It's a pen when you select the lighten tool.

**Comment by juliandescottes** [comment]
I agree, a different icon would be nice ! 

The current cursors are not inline with the rest of the application's design, so no need to spend too much time on the icon itself.
Maybe a modified version of the pen cursor, with a +/- box ? or a tiny yin-yang circle ?

**Comment by juliandescottes** [comment]
Same issue applies to other tools.

Right now, the pen cursor is shared by the pen, the lighting tool, the dithering tool and the stroke tool. Ideally each tool should have a different cursor.

Mouse cursors are defined in src/css/tools.css . Images are located in src/img/cursors

**Comment by epicabsol** [comment]
I gave this a go (#518), although I think that a redesign with a geometric, monochrome style would fit the current icon theme better.

**Comment by juliandescottes** [comment]
> although I think that a redesign with a geometric, monochrome style would fit the current icon theme better 

I agree ! But in the meantime, your current PR is a nice improvement.

**Comment by epicabsol** [comment]
Can this issue be closed, or are we waiting on a more stylish cursor set?

**Comment by juliandescottes** [comment]
No we're good :) let's close it.

I will open another issue to perform a global redesign of the cursor icon set (as you said they don't really match the current theme).
